net/qpid-proton/Portfile

```diff
@@ -19,14 +21,15 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.29.0
-revision            1
-
 PortGroup           cmake 1.1
```

Comment:
This too should go on top. Right after `PortSystem 1.0`

Reply:
@mf2k Thanks again for the feedback. I moved stuff as I think was required and currently 6 successful checks and 1 pending on travis so will check back to ensure that one turns out good as well.